### ISSUE TITLE
Keyboard focus color contrast with background color is 2.379:1 on Calander control and control present inside Calander popup: A11y_Azure Tools Storage Explorer_Manage Stored Access Policies_Nontext Contrast

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/AzureStorageExplorer#8411]
### Preflight Checklist

- [x] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [x] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [x] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.36.2

### Architecture

x64

### Platform

Windows

### OS Version

Windows11

## GitHub Tags:
#A11yMAS; #A11yTCS; #Win11; #DesktopApp; #A11ySev2; #BM_AzureToolsStorageExplorer_Win32_Dec2024; #Azure Tools Storage Explorer; #Devtools; #WCAG4.1.2; #FTP; 

## Repro Steps

1. Launch Azure Storage Explorer application.
2. TAB till "Account Management" in left Navigation and press ENTER key.
3. TAB till any "Subsection" for example 'azure tools Manual testing 1' and select it.
4. TAB till "Open explorer" button and press ENTER key
5. "Explorer" section will open. TAB till selected subscription and expand it
6. TAB till storage account and expand it.
7. TAB till any storage account and expand it.
8. TAB till "Blob Containers" and press ENTER key to expand it.
9. TAB till any "Blob container" and press ENTER key to select it now right click and press down arrow key to "Mange stored access polices" and press ENTER key.
10. TAB till "Add" button and press ENTER key.
11. TAB till "calendar" button and Press ENTER key
12. Observe that keyboard focus color contrast with background color is 2.379:1 on Calander control and control present inside Calander popup.

## Actual Experience
Keyboard focus color contrast with background color is 2.379:1 on Calander control and control present inside Calander popup.

## Expected Experience
Keyboard focus color contrast with background color should be equal or greater than 3:1.

## User impact
Low visibility user will face difficulty to see the keyboard focus.

### Actual Experience

ef

**Comment by InduPriya1805** [comment]
rev:inse

**Comment by kapilvaishna** [comment]
@craxal I have dropped mail to chromium edge team to log external bug, as per the external bug process, Once the external bug is fixed and closed, we will verify and close this issue. 

SL:External to chromium Edge:01-08-25

**Comment by kapilvaishna** [comment]
@craxal we got the response from external team for then issue is not repro for below version 

2nd issue Not Repro:
Win11: 
OS Version: 24H2 
OS Build: 26100.1742
Edge Canary Browser: 133.0.3061.0

![Image](https://github.com/user-attachments/assets/773f689e-d97e-43ec-afd0-58b5d621d1a0)

**Comment by kapilvaishna** [comment]
I have verified the issue in 1.37.0, it is reproducing

![Image](https://github.com/user-attachments/assets/eab94646-8071-4adc-8a47-2e32ede26477)


![Image](https://github.com/user-attachments/assets/c9c63302-6f8e-42a5-a96a-990359d2249f)


![Image](https://github.com/user-attachments/assets/c244b1d4-71cb-474f-9a95-9f0570c0866d)

**Comment by craxal** [comment]
@kapilvaishna Based on what you've told us, the bug is in Chromium and has been fixed in v133? That version is not yet available (it's still in Canary). The latest version of Electron (34) is using Chromium 132. We will not be able to fix this until a future Electron update. Therefore, we will not be able to fix this bug within the 60-day window.

**Comment by kapilvaishna** [comment]
@craxal ok once fix is available, please let us know to verify the issue.

**Comment by craxal** [comment]
@kapilvaishna Electron 35 is available, which has Chromium 134. However, the problem has not been resolved. Can you please post a link any external issue here for tracking?

**Comment by Yash14j** [comment]
Adding the tag #ResolvedByTT;

**Comment by kapilvaishna** [comment]
eDAD bug ID: 241387

**Comment by craxal** [comment]
Followed up with testers. External bug has been acknowledged. Waiting for links to those external bugs.

**Comment by craxal** [comment]
@kapilvaishna Any updates on the external bugs? We still don't have links or access to actually see what the current status is.

**Comment by jinglouMSFT** [comment]
@kapilvaishna Any updates on the external bugs? We don't have links or access to actually see what the current status is.

**Comment by kapilvaishna** [comment]
@jinglouMSFT @craxal For any bug external to Chromium team, eDAD don't log any external bug they sharing them through mail. There not tracing ID for us. I have asked eDAD if there is any update on issue.

**Comment by jinglouMSFT** [comment]
@kapilvaishna What's this you listed above>  eDAD bug ID: 241387

**Comment by kapilvaishna** [comment]
@jinglouMSFT According to the third-party external bug process, if a bug is identified as external to a third party, we report it to the enterprise Disability Answer Desk (eDAD) team, which I have shared (eDAD bug ID: 241387). The eDAD team is responsible for triaging the third-party bug, logging it in the third-party system, and providing a tracking ID to the product team. However, the eDAD team has stated that if a bug is external to Google, they do not log it as an external bug but instead share it via email.

**Comment by craxal** [comment]
Although we can't do anything about external components, we should at least ensure there's a note in our accessibility assessment/documentation.

**Comment by charan-kamasani** [comment]
Closing as not planned. This nontext-contrast (focus color) finding is in a third-party/external Calendar (date picker) control, which we do not own and cannot restyle to meet the 3:1 contrast requirement. Per our investigation, there is no Storage Explorer-side change available; we will note the external limitation in our accessibility assessment/documentation. Please reopen if the control becomes themeable or is replaced with one we control.

**Comment by craxal** [comment]
Reopening, as this is an accessibility bug, and accessibility testers need to close.

**Comment by kapilvaishna** [comment]
Storage container is only available for one subscription among all subscription and for other subscription I don't have access to create storage account. For available Storage container I am not getting the "Mange stored access polices" control as per repro step 9.

<img width="1280" height="768" alt="Image" src="https://github.com/user-attachments/assets/cfbdf59e-0a4f-469d-afd5-ad0aabf9d9fb" />

**Comment by craxal** [comment]
@kapilvaishna Do you have access to account keys in that storage account? Either get assigned an RBAC role with `listkeys` (Contributor / Storage Account Contributor / Storage Account Key Operator), or attach the account directly with a connection string or account key. Then the item should appear on the container's right-click menu.

**Comment by craxal** [comment]
@kapilvaishna Any updates? Were you able to unblock?
